src/Compilers/Core/Portable/Compilation/TypeInfo.cs

```diff
@@ -17,12 +17,16 @@ public struct TypeInfo : IEquatable<TypeInfo>
         /// </summary>
         public ITypeSymbol Type { get; }
 
+        public Nullability TypeNullability { get; }
```

Comment:
Doc comment on new APIs would be helpful.

Reply:
The name "Type" as part of this API probably doesn't make sense.

Reply:
I included `Type` to give the user something to key off of.

---
In reply to: [205845601](https://github.com/dotnet/roslyn/pull/28860#discussion_r205845601) [](ancestors = 205845601)